data/org.freedesktop.portal.GlobalShortcuts.xml

```diff
@@ -88,6 +82,30 @@
             </para></listitem>
           </varlistentry>
         </variablelist>
+
+        Shortcuts contain the following properties:
+        <variablelist>
+          <varlistentry>
+            <term>trigger s</term>
```

Comment:
Should this ever be something passed from the app? I assume it's rather something that is always set by the portal implementation, and handed to the app so it can be presented in some way.

Reply:
Now that I look into it... I wonder what's that even supposed to do. I suppose this was meant to be some sort of "title" or "name" of the shortcut, like "Create Source" or "Hide Something"?

Reply:
@aleixpol could you please clarify?

Reply:
Isn't it meant to let the application present in e.g. some configuration view what the global keybinding was bound to?

Reply:
The app provides a shortcut as specified by:
https://github.com/freedesktop/xdg-specs/blob/master/shortcuts/shortcuts.txt

Then the portal replies with a trigger description to be rendered on the app to help the user.

(not sure if this is what you were asking)

Reply:
I think the question is, why is there:

```
            <term>trigger s</term>
            <listitem><para>
              User-readable text describing how to trigger the shortcut for the
              client to render.
            </para></listitem>
```

when there already is

```
            <term>shortcut s</term>
            <listitem><para>
              The suggested shortcut trigger, defined as described by the "shortcuts"
              XDG specification. Optional.
            </para></listitem>
```

Reply:
What @jadahl said. ^

Reply:
I'm not sure that separating `trigger` from `shortcut` is a good idea. Portal implementations could derive a user-visible string from a shortcuts, couldn't they?

Reply:
I'm concerned with the fact that we can't validate that `trigger` is equal to `shortcut`

Reply:
It's perfectly valid that it isn't. Big part of this whole process is for the portal to show a dialog that will allow the user to change the shortcut eventually. Maybe we can rename `shortcut` to `default_shortcut`?

Reply:
Maybe `shortcut_hint`?

Reply:
My suggestion here is to split up the vararg specification to two different ones: one "passed to portal", that contains `shortcut`/`shortcut_hint`/`default_shortcut`/whatever, and another that the portal passes to the app, that contains `trigger`.

That makes it clear that the app passes the suggested shortcut according to the machine readable format, and that the portal passes the trigger in a human readable format back with the chosen trigger.

Reply:
> My suggestion here is to split up the vararg specification to two different ones:...

Makes sense to me. This is something I also considered doing, did not do it because I feared it might overcomplicate the spec, but it clearly still is confusing...

Reply:
Sounds good